ape/ape.lds

```diff
@@ -349,9 +349,30 @@ SECTIONS {
     HIDDEN(_etext = .);
     PROVIDE_HIDDEN(etext = .);
 /*END: Read Only Data (only needed for initialization) */
-/*END: Read Only Data */
   } :Rom
 
+/* initialization image for thread-local storage, this is copied */
```

Comment:
This is the hardest part of the linker script. Congratulations on successfully editing it. I've spent many hours scratching my head about how to make TLS work with the linker.